docs/website/contents/for-developers/OperationalCertificateIssueNumber.md

```diff
@@ -0,0 +1,81 @@
+
+Completely orthogonally to OCINs, each HotKey is confined to one interval of slots, the {KesPeriod}, which is ~90 days on Cardano.
+This achieves _forward-security_ ([Wikipedia](https://en.wikipedia.org/wiki/Forward_secrecy)) with respect to long-term time scales without requiring SPOs to access their ColdKey unbearably often (recall that it's physically offline).
+
+If Cardano did not involve OCINs, then an adversary would benefit from an acquired HotKey until its KesPeriod expired, which could be up to 90 days.
+On the other hand, an SPO---especially one with a lot of delegated stake---can increment the OCIN much sooner than 90 days after an attack (assuming they notice it).
+
+When issuing a new HotKey merely because the current one expired, an SPO does not need to increment their OCIN.
+Generally, there's no downside if they do so, other than increasing the likelihood of some awkward situations described in the next section---even then it's a risk that arises only once per 90 days per stake pool.
+
+*Remark*.
+[The current Cardano documentation](https://developers.cardano.org/docs/operate-a-stake-pool/cardano-key-pairs/#stake-pool-cold-keys) disagrees, stating it's crucial to increment when rotating.
+Moreover, the relevant CLI tool always increments ([here](https://github.com/IntersectMBO/cardano-api/blob/4a6ce60b0028e3062d666980574aebf6acfee9b3/cardano-api/src/Cardano/Api/Certificate/Internal/OperationalCertificate.hs#L134) and [here](https://github.com/IntersectMBO/cardano-cli/blob/2124f5ab210ef57a6ed25c8cd383b36f927b6415/cardano-cli/src/Cardano/CLI/EraIndependent/Node/Run.hs#L292-L304)).
+We suspect either a past miscommunication or else those authors considered the nuance to not be worth the risk, favoring the conservative/robust approach of always incrementing.
+
+## Interaction with Short Forks
+
+Different forks might disagree on some ColdKeys's Z.
+In the most common circumstance, this makes no difference to the SPO.
+If X issues a header with an incremented Y+1 that doesn't end up on the honest chain (ie is orphaned), they'll simply issue their next header with the same Y+1 on the winning fork.
+
+On the other hand, if unusual circumstances cause a party X to increment its OCIN multiple times within the same stability window (12 to 36 hr on Cardano, shorter when Peras succeeds), then it's possible that a short fork might discard N>1 of those increments.
+In that case, X's next header on the winning fork cannot be its greatest ever OCIN (ie Z on the losing short fork), but rather N-1 less than that.
+Thus there will be an extended interval during which the adversary can abuse X's elections, if X actually did need to increment their OCIN N times.
+
+The unusual scenario might be visualized as follows, where the chain on the left is the losing fork and the chain on the right is the winning fork.
+The unfortunate property here is that X has issued a header with Y=1 _after_ having issued a header with Y=2.
```

Comment:
```suggestion
The unfortunate property here is that X has issued a header with Y=1 _after_ having issued a header with Y=2.
```
I don't get this bit. Are we talking about issuing a header on different forks? But in any case, how can the same party issue `Y=1` after increasing the opcert number?

Reply:
The only increased their OCIN on the _losing chain_ not on the _winning chain_. Now that they've seen the outcome of the short fork, they're now contributing to the winning chain. They cannot use their latest OCIN, since that's only valid on the losing fork. So they awkwardly have to use one of their previous OCINs that wasn't increased in order to contribute to the best chain they can (as Praos requires), since Z was never incremented (or was incremented fewer times) on the fork that won the contest.